app/src/main/kotlin/org/wordpress/aztec/demo/MainActivity.kt

```diff
@@ -340,11 +340,11 @@ class MainActivity : AppCompatActivity(),
 
         // initialize the text & HTML
         if (!isRunningTest) {
-            aztec.sourceEditor.displayStyledAndFormattedHtml(EXAMPLE)
+            aztec.sourceEditor?.displayStyledAndFormattedHtml(EXAMPLE)
         }
 
         if (savedInstanceState == null) {
-            aztec.visualEditor.fromHtml(aztec.sourceEditor.getPureHtml())
+            aztec.visualEditor.fromHtml(aztec.sourceEditor?.getPureHtml()!!)
```

Comment:
This may crash if we don't initialize the source editor (the effect of using `?.` is that a `null` value is returned if  `sourceEditor` is `null`). Since this is just a demo and we know that we will have a source editor, this fine. But using `!!` is dangerous and it should only be used when you are certain the expression can never be `null`.